CHANGELOG.rst

```diff
@@ -14,6 +14,10 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+[0.12.0] - 2021-09-23
```

Comment:
I think this PR kinda makes the version from 0.x to 1.x. It's like a major change to the repo and it could be a breaking change in openedx.